File: viewer-dotnet-winform-vb/MainForm.vb
Public Class MainForm
    Private Sub btn_CustomSearchDialog_Click(sender As Object, e As EventArgs) Handles btn_CustomSearchDialog.Click
        Dim customSearchDialog As CustomSearchInterface = New CustomSearchInterface()
        customSearchDialog.ShowDialog()
    End Sub

    Private Sub btn_CustomToolStrip_Click(sender As Object, e As EventArgs) Handles btn_CustomToolStrip.Click
        Dim customToolStrip As CustomToolStrip = New CustomToolStrip()
        customToolStrip.ShowDialog()
    End Sub

    Private Sub btn_HiddenToolbars_Click(sender As Object, e As EventArgs) Handles btn_HiddenToolbars.Click
        Dim hiddenToolbars As HiddenToolbars = New HiddenToolbars()
        hiddenToolbars.ShowDialog()
    End Sub

    Private Sub btn_ParallelView_Click(sender As Object, e As EventArgs) Handles btn_ParallelView.Click
        Dim parallelView As ParallelView = New ParallelView()
        parallelView.ShowDialog()
    End Sub

    Private Sub btn_PDFViewerCustomizations_Click(sender As Object, e As EventArgs) Handles btn_PDFViewerCustomizations.Click
        Dim pDFViewerCustomizations As PDFViewerCustomizations = New PDFViewerCustomizations()
        pDFViewerCustomizations.ShowDialog()
    End Sub

    Private Sub btn_PDFViewerLocalization_Click(sender As Object, e As EventArgs) Handles btn_PDFViewerLocalization.Click
        Dim pDFViewerLocalization As PdfViewerLocalization = New PdfViewerLocalization()
        pDFViewerLocalization.ShowDialog()
    End Sub

    Private Sub btn_SimplePDFViewer_Click(sender As Object, e As EventArgs) Handles btn_SimplePDFViewer.Click
        Dim simplePDFViewer As SimplePdfViewer = New SimplePdfViewer()
        simplePDFViewer.ShowDialog()
    End Sub

    Private Sub btn_SplitControls_Click(sender As Object, e As EventArgs) Handles btn_SplitControls.Click
        Dim splitControls As SplitControls = New SplitControls()
        splitControls.ShowDialog()
    End Sub
End Class

File: viewer-dotnet-winform-cs/MainForm.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace viewer_dotnet_winforms_cs
{
    public partial class MainForm : Form
    {
        public MainForm()
        {
            InitializeComponent();
        }

        private void btn_CustomSearchDialog_Click(object sender, EventArgs e)
        {
            CustomSearchInterface customSearchDialog = new CustomSearchInterface();
            customSearchDialog.ShowDialog();
        }

        private void btn_CustomToolStrip_Click(object sender, EventArgs e)
        {
            CustomToolStrip customToolStrip = new CustomToolStrip();
            customToolStrip.ShowDialog();
        }

        private void btn_HiddenToolbars_Click(object sender, EventArgs e)
        {
            HiddenToolbars hiddenToolbars = new HiddenToolbars();
            hiddenToolbars.ShowDialog();
        }

        private void btn_ParallelView_Click(object sender, EventArgs e)
        {
            ParallelView parallelView = new ParallelView();
            parallelView.ShowDialog();
        }

        private void btn_PDFViewerCustomizations_Click(object sender, EventArgs e)
        {
            PdfViewerCustomizations pDFViewerCustomizations = new PdfViewerCustomizations();
            pDFViewerCustomizations.ShowDialog();
        }

        private void btn_PDFViewerLocalization_Click(object sender, EventArgs e)
        {
            PdfViewerLocalization pDFViewerLocalization = new PdfViewerLocalization();
            pDFViewerLocalization.ShowDialog();
        }

        private void btn_SimplePDFViewer_Click(object sender, EventArgs e)
        {
            SimplePdfViewer simplePDFViewer = new SimplePdfViewer();
            simplePDFViewer.ShowDialog();
        }

        private void btn_SplitControls_Click(object sender, EventArgs e)
        {
            SplitControls splitControls = new SplitControls();
            splitControls.ShowDialog();
        }
    }
}
